ShowMyCursedRealm/room.aadl

```diff
@@ -1,45 +1,94 @@
 package room
 public
+with threads;
+with deployment;
+	
 	
 	device camera
 		features
-			video: out data port;
-			camera_on: in event port;
+			video_stream: out data port threads::Video_Stream;
```

Comment:
Dobrze myślę, że `threads::Viedeo_Stream` - to typ danych który jest przekazywany przez ten port?

Reply:
Te typy są TODO, w sensie są inaczej nazwane, ale w zasadzie wszystkie mają representation integer raczej, to trzeba ustalić tak, żeby miało sens z kodem